R/data.R

```diff
@@ -69,7 +109,44 @@
 #' @source vignette_data.R
 "hosp_data"
```

Comment:
Should probably document this dataset and "hosp_data_eval" as being derived from the spatially correlated model.